src/backend/src/utils/validation.utils.ts

```diff
@@ -0,0 +1,5 @@
+
+
+export const intMinZero = (validationObject: any): any => {
```

Comment:
validationObject should not be type `any` or return `any` either. check out what the types of the body('asdf') things are in the routes files and put that here